frontend/public/components/factory/list.tsx

```diff
@@ -403,6 +403,8 @@ Rows.propTypes = {
   mock: PropTypes.bool,
 };
 
+const skeleton = <div className="loading-skeleton--table" />;
```

Comment:
Perhaps this should be named `SkeletonTable` since there are other skeletons?

Reply:
changed to `skeletonTable`